doc/qbk/02_2_strings.qbk

```diff
@@ -12,22 +13,58 @@
 [section:strings Using Strings]
 
 Modifiable sequences of characters are represented using instances of type
-[link json.ref.boost__json__string `string`].
-These strings have an interface almost identical to `std::string`, and
-operate the same way, with these variations:
+__string__.
+These strings have an modified `string_view` based version of the `std::string` interface, and
+operate the same way, except that:
 
-* __string__ is not a class template.
+* __string__ is not a class template,
+
+* __string__ only uses `char` as its character type,
+
+* redundant overloads for string operations have been replaced with a `string_view` based interface,
+
+* access to characters in the range `[size(), capacity()]` is permitted, and
```

Comment:
This should be a half-open interval `[ ... )`

Reply:
Fixed.

Reply:
(off-topic) I'm new here, what is the workflow?
- @vinniefalco requests the change
- @sdkrystian makes the change
- who hits "Resolve conversation"? @sdkrystian after fixing or @vinniefalco after verifying it's been fixed?

Reply:
I don't use any of that I just leave individual comments.

Reply:
All are individual comments but if thread is not resolved, not closed, it's dangling post PR update and confusing reviews, what's been addressed.